optimizer.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+function optimizer (data) {
+    const objArray = Object.entries(data);
+    const modArray = objArray.map(entryPair => 

```

Comment:
What do you say if I'd suggest you this destructuring:
```
const modArray = objArray.map(([key, value])=>
```

Reply:
I would say that the previous version did not work - I hadn't used the test sample you provided and therefore hadn't applied Number() to the value. Shame.
The proposed option is nice and simpler. Rewritten.